app/controllers/declaration/SubmissionController.scala

```diff
@@ -84,6 +85,15 @@ class SubmissionController @Inject() (
         )
   }
 
+  def cancelAmendment(decId: Option[String]): Action[AnyContent] = (authenticate andThen verifyEmail) { implicit request =>
+    decId match {
+      case Some(id) if declarationAmendmentsConfig.isEnabled =>
+        Redirect(routes.SubmissionController.submitAmendment(Some("cancel"))).addingToSession((ExportsSessionKeys.declarationId, id))
+      case _ =>
+        Redirect(controllers.routes.SignOutController.userSignedOut)
```

Comment:
Are the requirements actually asking to redirect to the SignOutController if the flag is disabled? Or should the redirection be to the root controller?

Reply:
You're right, that's been left in there from development